static/js/base.js

```diff
@@ -378,6 +379,18 @@ function log(text) {
     const time = '[' + ((hour < 10) ? '0' + hour: hour) + ':' + ((minutes < 10) ? '0' + minutes: minutes) + '] ';
 
     let consoleLog = document.getElementById("console");
-    consoleLog.innerHTML += time + text + "<br>";
+    if (text.includes("Python exception:")) {
+        pythonError = true;
+    }
+    if (pythonError) {
+        consoleLog.innerHTML += "<span style= \"color: red;\">" + time + text + "</span>" + "<br>";
```

Comment:
I would slightly prefer this to be single quotes outside and double quotes inside